src/swsssdk/configdb.py

```diff
@@ -117,16 +123,30 @@ def __typed_to_raw(self, typed_data):
                 raw_data[key] = value
         return raw_data
 
+    @staticmethod
+    def serialize_key(key):
+        string_types = str if sys.version_info[0] == 3 else basestring
+        return key if isinstance(key, string_types) else '|'.join(key)
+
+    @staticmethod
+    def deserialize_key(key):
+        tokens = key.split('|')
+        return tuple(tokens) if len(tokens) > 1 else tokens[0]
```

Comment:
>eturn tuple(tokens) if len(tokens) > 1 else tokens[0] [](start = 9, length = 53)

not readable, I think we should try to avoid to keep the code more readable.